### ISSUE TITLE
Staging Review finding: "Learn more" and "Apply now" links don't have unique accessible names

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#93692]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Transition Experience
**Product:** Transition Experience MVP v1
**Feature:** V1 Transition Experience

## Findings details
**VA.gov Experience Standard - issue:** [User can't identify an element or its state.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

When reviewing benefits to explore, each benefit has a "Learn more" link and some benefits have an "Apply now" link. The text of each link is the same across each benefit, and no additional ARIA text is provided to differentiate the links.

It is very difficult to determine which link goes with which benefit without the visual design cues --- for instance, if you're a screen reader user. Screen reader users often skim a page by tabbing through each focusable element or by having their screen reader announce a list of all of the links on the page. Users interacting with the page in this way will hear "Learn more" a lot, without a lot of clarity about what they'd be learning about.

<details><summary>Link, screenshot or steps to recreate</summary>

Example of a benefit listed with both "Learn more" and "Apply now" links:

![GI Bill education benefit card, with text describing the program and two action links: Learn more and Apply now.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/82863-Brian-2.png?raw=true)

</details>

## Recommended action

Each link must have a unique accessible name. The easiest way to do this is to make the link text unique: "Learn more about the GI Bill" and "Apply now for the GI Bill" or similar. The shortest path to accessibility is often visible text for everyone.

If updating the link text isn't practical for design reasons, the [`va-link-action` component](https://design.va.gov/storybook/?path=/docs/components-va-link-action--docs) supports a `label` prop that populates an `aria-label` on the link. An `aria-label` replaces the visible text in the browser's accessibility tree, meaning screen readers will announce the `aria-label` instead of the visible text. Something like `<va-link-action href="https://www.va.gov/education/about-gi-bill-benefits/" text="Learn more" label="Learn more about the GI Bill" type="secondary" class="hydrated"></va-link-action>` would be appropriate.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** [WCAG 4.1.2 Name, Role, Value - Level A](https://www.w3.org/TR/WCAG21/#name-role-value)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by nkontrabecki** [comment]
Nicole confirmed fixed and tested in Sprint 13. 10/7/2024

**Comment by nkontrabecki** [comment]
Completed in Sprint 18.
